### PR TITLE
Todo list using react

### PR DESCRIPTION
**Pull request by sediqullahbadakhsh**
in this milestone, I have created a react app using routers, hooks, and functional componenets.

### CODE REVIEW

**Review by leopiresgoss — CHANGES_REQUESTED** [review]
Good job @sediqullahbadakhsh :1st_place_medal: 

What I liked about your project:
:heavy_check_mark:  Well structured and organized
:heavy_check_mark: No linter issues
:heavy_check_mark: Great usage of gitflow 
:heavy_check_mark: Great usage of Hooks

However, I have some suggestions for you in order to improve your project:

- Your todo app is not working, I couldn't add a task and it is throwing those errors:
image

- Kindly, update your readme file in a professional way.

Happy coding :slightly_smiling_face:

File: src/components/InputTodo.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    } else {
+      // alert('Please write item');
+      setInputText({ message: 'Please write something' });
+    }
+  };
+
+  return (
+    <div>
+      <form onSubmit={handleSubmit} className="form-container">
+        <input
```
Comment (Have-Samuel) [inline src/components/InputTodo.js:33]: Nice code :100:

File: src/components/TodoContainer.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { Route, Routes } from 'react-router-dom';
+import Header from './Header';
+import InputTodo from './InputTodo';
+import TodosList from './TodosList';
+import About from '../pages/About';
+import NotMatch from '../pages/NotMatch';
+import Navbar from './Navbar';
```
Comment (Have-Samuel) [inline src/components/TodoContainer.js:9]: Good job done with the modules :+1:

File: src/components/TodoItem.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+  const handleEditing = () => {
+    setEditing(true);
+  };
+
+  const handleUpdatedDone = (event) => {
+    if (event.key === 'Enter') {
+      setEditing(false);
+    }
+  };
```
Comment (Have-Samuel) [inline src/components/TodoItem.js:19]: Good job done with the functions :100:

**Review by Have-Samuel — COMMENTED** [review]
Good job done with the code :+1: , I just left a few comments below, please try to look into them so that the code becomes more clear.

And feel free to reach out to me in case of any issues.

**Review by Have-Samuel — APPROVED** [review]
Good Job done, Clean code :100:

**Pull request merged**